README.md

```diff
@@ -65,19 +65,33 @@ To this:
 }
 ```
 
-_The `betterScripts` script definition can either be a string or sub-object with `command` and `env` attributes. Values defined in the `env` block will override previously set environment variables._
+_The `betterScripts` script definition can either be a string or sub-object with `command`, `env`, and `envpath` attributes. Values defined in the `env` block will override previously set environment variables._
 
 # .env File
 
-If you have an `.env` file in your project root it will be loaded on every command.
+If you have an `.env` file in your project root it will be loaded on every command. Path to the `.env` file can be further specified with the CLI argument.
 
 ```
 NODE_PATH=./:./lib
 NODE_ENV=development
 PORT=5000
 ```
 
-_Environment variables defined in the `betterScripts` script definition will take precedence over `.env` values._
+_Path to the `.env` file specified via `envpath` attribute in the `betterScripts` script definition will take precedence over default path or path passed with the CLI argument._
```

Comment:
Let's simplify this explanation a bit…thoughts on removing the example and replacing with the below statement?

> _Environment variables defined using the `env` object in package.json will overwrite values loaded from an `.env` file. When the `envpath` option is defined in package.json, the root level env file or `--path` CLI option will not be loaded._